src/targets/gpu/lowering.cpp

```diff
@@ -253,8 +253,10 @@ struct miopen_apply
             assert(refs.size() == 2);
             auto output = insert_allocation(ins, ins->get_shape());
             refs.push_back(output);
+
 #if MIGRAPHX_USE_HIPBLASLT
-            if(enabled(MIGRAPHX_DISABLE_HIPBLASLT_GEMM{}) or not hipblaslt_supported())
+            if((string_value_of(MIGRAPHX_SET_GEMM_PROVIDER{}) == "rocblas") or
```

Comment:
the previous code didn't have this either, but could you add a comment mentioning this is setting to use rocblas if user specified or hardware limited?
```suggestion
            // check if user explicitly sets rocblas as gemm provider, or if hardware cannot support hipblaslt
            if((string_value_of(MIGRAPHX_SET_GEMM_PROVIDER{}) == "rocblas") or
```